features/admin.validation.v1/pages/validation-config-edit.tsx

```diff
@@ -1090,9 +1066,7 @@ export const ValidationConfigEditPage: FunctionComponent<MyAccountSettingsEditPa
                                 return t("common:minValidation", { min });
                             }
 
-                            const max: number = ValidationConfigConstants
-                                .VALIDATION_CONFIGURATION_FORM_FIELD_CONSTRAINTS
-                                .PASSWORD_MAX_VALUE;
+                            const max: number = maxPasswordLenghtLimit;
```

Comment:
Instead of defining new variable, we can use the same `maxPasswordLengthLimit` variable in place of `max`?

Reply:
Fixed by https://github.com/wso2/identity-apps/pull/7185/commits/99ed0d0d4a9a54e5599b00b8ade180816d2daea4